scripts/install_backend_python_libs.py

```diff
@@ -78,7 +78,8 @@ def normalize_python_library_name(library_name):
     # same way (e.g import google.api.core) and if pip allowed a scenario where
     # both versions were installed, then there would be ambiguities in the
     # imports. For this reason, it is safe to disambiguate the names by removing
-    # the suffix.
+    # the suffix. We have also implemented a backend test to ensure that all
```

Comment:
As mentioned before, include a reference to the test (specify the name + file location).

Reply:
Oh I see, sorry I shouldn't have assumed that this implies that it would be in the same test file.